cmake-modules/AzureVcpkg.cmake

```diff
@@ -18,13 +18,50 @@ macro(az_vcpkg_integrate)
   endif()
 endmacro()
 
-macro(az_vcpkg_export targetName macroNamePart dllImportExportHeaderPath)
+macro(az_vcpkg_portfile_prep targetName fileName contentToRemove)
+  # with sdk/<lib>/vcpkg/<fileName>
+  file(READ "${CMAKE_CURRENT_SOURCE_DIR}/vcpkg/${fileName}" fileContents)
+
+  # Windows -> Unix line endings
+  string(FIND fileContents "\r\n" crLfPos)
+  string(REPLACE "\r\n" "\n" fileContents ${fileContents})
```

Comment:
Put this replace under an `if (crLfPos GREATER -1)` block just like below. If we don't find any "\r\n", we don't need to do a replace.